libs/tasks/feature/src/lib/tasks-feature/tasks-feature.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-template #todoHeader>
+  <h1>TODO List</h1>
+  <button mat-raised-button color="primary" (click)='openCreateDialog()'>New task</button>
+</ng-template>
+<todo-workspace-tasks-list [headerTemplate]='todoHeader'
```

Comment:
Maybe a better way to provide HTML to the component is providing via `ng-conent`

Reply:
ok